src/components/DiagnosticCard/DiagnosticCard.scss

```diff
@@ -5,18 +5,18 @@
     padding: 16px;
     padding-bottom: 28px;
 
-    border: 1px solid var(--yc-color-line-generic);
+    border: 1px solid var(--g-color-line-generic);
     border-radius: 8px;
     background-color: transparent;
 
     &_active {
-        border-color: var(--yc-color-base-info-heavy);
-        background-color: var(--yc-color-base-selection);
+        border-color: var(--g-color-base-info-medium);
```

Comment:
Color palette was changed, new `-heavy` is too bright, so replaced it with `-medium`